### PR TITLE
stop replace all errors

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
https://sentry.io/share/issue/c55c9e1c8c304af2a4b54177132c94f4/
## Description
Use old school replace instead of replace all for old browser compatibility

### CODE REVIEW

File: web/src/features/panels/zone/Attribution.tsx [hunk 1]
```diff
@@ -12,9 +12,7 @@ export function removeDuplicateSources(source: string | undefined) {
     ...new Set(
       source
         .split('","')
-        .flatMap((x) =>
-          x.split(',').map((x) => x.replaceAll('\\', '').replaceAll('"', ''))
-        )
+        .flatMap((x) => x.split(',').map((x) => x.replace(/\\/g, '').replace(/"/g, '')))
```
Comment (VIKTORVAV99) [inline web/src/features/panels/zone/Attribution.tsx:15]: Just to make sure this still replaces all instances and not just the first one right?

Comment (tonypls) [reply web/src/features/panels/zone/Attribution.tsx:15]: Yeah, that's what the /g does :)

Comment (VIKTORVAV99) [reply web/src/features/panels/zone/Attribution.tsx:15]: Thought so just wanted to make sure. 👍🏼

**Review by madsnedergaard — APPROVED** [review]
Nice catch! Ideally we should get Vite legacy plugin working so we don't have to think too much about this stuff when developing :)

**Pull request merged**